### DISCUSSION TITLE
sherlock stuck in a step

### DISCUSSION THREAD

**Discussion by BIOHS** [sherlock-project/sherlock#2183] [Q&A] [answered]
![pic](https://github.com/sherlock-project/sherlock/assets/154678757/372f3f8e-431c-4f1d-b266-af86ded1269e)
What I do after?
The yellow command isn't working with me, how can I solve this issue?
I have search in YouTube, google but I've didn't arrive to answer

**Comment by matheusfelipeog** [comment, marked as answer]
The `requirements.txt` file is no longer used in recent versions here on GitHub.

If you just want to use Sherlock, install it using `pip`:

```zsh
pip install sherlock-project
```

or `pipx`:

```zsh
pipx install sherlock-project
```

See other ways to install it [here](https://sherlockproject.xyz/installation):

**Reply by BIOHS** [reply]
Thanks!
but I've found another problem lol If you can help
why the users doesn't appeared?
![pic](https://github.com/sherlock-project/sherlock/assets/154678757/4386733c-fd74-407b-a3b8-9a4fa397a8b5)

**Reply by matheusfelipeog** [reply]
This way you are trying to run the Python package that is in the sherlock directory.

If you installed it as I mentioned, just use the command `sherlock`:

```zsh
sherlock parlin
```

You don't need to have the repository cloned on your machine to use the Sherlock project.

**Reply by BIOHS** [reply]
Done. 
Thank you very much for your help
